stdlib/REPL/src/REPL.jl

```diff
@@ -545,6 +545,17 @@ display(d::REPLDisplay, x) = display(d, MIME("text/plain"), x)
 
 show_repl(io::IO, mime::MIME"text/plain", x) = show(io, mime, x)
 
+# new method for AbstractChar
+function show_repl(io::IO, mime::MIME"text/plain", c::AbstractChar)
+    show(io, mime, c) # Call the original Base.show
+    # Check for LaTeX/emoji alias and print if found and using symbol_latex which is used in help?> mode
+    latex = symbol_latex(string(c))
+    if !isempty(latex)
+        print(io, ", input as ")
+        printstyled(io, latex, color=:cyan)
```

Comment:
```suggestion
        printstyled(io, latex, "<tab>"; color=:cyan)
```
Including the note of the tab key brings this closer in line with how `repl_latex` displays this information.

Reply:
Thanks! I've added the <tab> suffix to the printstyled call in the latest commit to better match the help mode display. I've also added tests covering characters with and without aliases in stdlib/REPL/test/repl.jl as requested.